data/widgets/card/list.ui

```diff
@@ -92,6 +92,18 @@
                 </style>
               </object>
             </child>
+            <child>
+              <object class="GtkFrame" id="context-frame">
```

Comment:
No particular reason really, except that this is what we do for the others cards e.g. https://github.com/endlessm/eos-knowledge-lib/blob/master/data/widgets/card/defaultFamily.ui#L31

So I am just duplicating it for consistency.